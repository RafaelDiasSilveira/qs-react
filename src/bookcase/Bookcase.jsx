import React from 'react';

import Header from './BookcaseHeader';
import Navigation from './BookcaseNavigation';

const Bookcase = () => (
  <div>
    <Header />
    <Navigation />
  </div>
);

export default Bookcase;
