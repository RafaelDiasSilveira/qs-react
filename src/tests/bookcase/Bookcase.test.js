import React from 'react';
import { shallow } from "enzyme";

import Bookcase from "../../bookcase/Bookcase";

test('should render Bookcase correctly', () => {
  const wrapper = shallow(<Bookcase />);

  expect(wrapper).toMatchSnapshot();
});
