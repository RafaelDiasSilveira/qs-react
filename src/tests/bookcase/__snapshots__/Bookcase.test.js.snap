// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Bookcase correctly 1`] = `
<div>
  <Component />
  <Component />
</div>
`;
